extensions/wikia/AdEngine/js/spec/lookup/rubicon/rubiconVulcan.spec.js

```diff
@@ -221,6 +221,6 @@ describe('ext.wikia.adEngine.lookup.rubicon.rubiconVulcan', function () {
 
 		vulcan.call();
 
-		expect(vulcan.getSlotParams('TOP_LEADERBOARD')).toEqual({});
+		expect(vulcan.getSlotParams('TOP_RIGHT_BOXAD')).toEqual({});
```

Comment:
:trollface: what if we add medrec to Vulcan calls in future? I'm kidding. I think it's fine. Once we add (if we add) there medrec the one working on it will have to update unit tests which is an interesting practice :japanese_ogre: